java/src/main/java/com/google/oak/client/OakClient.java

```diff
@@ -42,19 +51,23 @@ public class OakClient<T extends Transport> implements AutoCloseable {
    * @return an instance of the {@code OakClient} wrapped in a {@code Result}
    */
   public static <E extends EvidenceProvider & Transport, V extends AttestationVerifier>
-      Result<OakClient<E>, Exception> create(E transport, V verifier) {
+      Result<OakClient<E>, Exception> create(
+          E transport, V verifier, Clock clock, ReferenceValues referenceValues) {
```

Comment:
I think instead of passing in reference values we can expect that reference values are already a part of the verifier. So that the `verify` function on the verifier only accepts time, evidence and endorsements.

Because for example for the insecure verifier we don't need reference values